webplugin/template/chat.html

```diff
@@ -53,15 +53,20 @@
                         if (window.KommunicateGlobal && preChatLauncher) {
                             var intervalForPreChatLead = setTimeout(function () {
                                 preChatLauncher.click()
+                                iframe.classList.add("change-kommunicate-iframe-height");
+                                var preChatLeadScreenId = window.KommunicateGlobal.window.document.getElementById("km-chat-login-modal")
+                                var cssForPreChatLeadScreen = "#km-chat-login-modal{height:100%!important;width:100%!important;right:0!important;bottom:0!important;max-height: 100%!important;}"
+                                preChatLeadScreenId && Kommunicate.customizeWidgetCss(cssForPreChatLeadScreen);
                                 window.KommunicateGlobal.window.document.getElementById("km-modal-close").style.display = "none"
-                            }, 500);
+                            }, 100);
```

Comment:
what is the purpose of 100 here

Reply:
I am checking as soon as this item is available - if it check it on 500ms interval - I see a delay jisse icon dikh rha hai

Reply:
you mean after 100ms you are checking something?

Reply:
yes - 
i am checking for the id so that I can show the prechatlead window

var preChatLauncher = window.KommunicateGlobal.window.document.getElementById("km-anonymous-chat-launcher")

Reply:
what is the guarantee that it will be done within 100 ms? what will happen if this assumption goes wrong for any reason?

Reply:
it will check every 100ms second